src/definitions/modules/search.js

```diff
@@ -337,7 +337,7 @@ $.fn.search = function(parameters) {
               apiSettings = {
                 debug             : settings.debug,
                 on                : false,
-                cache             : true,
```

Comment:
Why was this changed?

Reply:
I think the issue is I branched off master rather than next